src/stores/account.ts

```diff
@@ -61,14 +98,117 @@ export default class AccountStore implements IAccountStore {
   };
 
   @action
-  changeAddress = (index: number) => {
-    if (index >= 0 && index < this.addresses.length) {
-      this.selectedAddress = this.addresses[index];
+  setActivationKey = (activationKey: string) => {
+    this.activationKey = activationKey;
+  };
```

Comment:
> Question need to be answered : Is accountStore is too huge? we're using store as some kind of singleton instance, is too many functions focused on store okay?

In the above context, should `setActivationKey` be separated as another store?

Reply:
Is a bit complicated but we need ActivationKey in usually accountStore-called environment. so I'm going to leave it now, but your point also valid.